src/test/java/IntegrationTest/PersonControllerIntegrationTestPath.java

```diff
@@ -0,0 +1,54 @@
+package IntegrationTest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import springboot.BootApplication;
+import springboot.Controller.PersonController;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@WebAppConfiguration
+@ContextConfiguration(classes = BootApplication.class)
+public class PersonControllerIntegrationTestPath {
+
+    @Autowired
+    PersonController personController;
+
+    @Autowired
+    private WebApplicationContext wac;
+
+    private MockMvc mockMvc;
+
+    @Before
+    public void setup() {
+        this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
+    }
+
+    @Test
+    public void testLoadAddPersonPage() throws Exception {
+        this.mockMvc.perform(get("/person/addPersonPage"))
+                .andExpect(status().isOk())
+                .andExpect(view().name("addPersonPage"))
+                .andExpect(model().attributeExists("person"))
+                .andReturn();
+    }
+
+    @Test
+    public void testLoadPersonListPage() throws Exception {
+        this.mockMvc.perform(get("/person/personListPage"))
```

Comment:
url should be 'person/all'

Reply:
Done on local, to commit

This is a tracking comment